src/utils/getAssetsFilters.jsx

```diff
@@ -0,0 +1,23 @@
+import ASSET_TYPES from '../data/constants/assetTypeFilters';
+
+export const getSelectedFilters = (filters) => {
```

Comment:
This returns an array of the filters keys that are selected (i.e. whose values in the filters object are true).